modules/core/arrow-typeclasses/src/main/kotlin/arrow/typeclasses/MonadError.kt

```diff
@@ -16,15 +16,23 @@ interface MonadError<F, E> : ApplicativeError<F, E>, Monad<F> {
 
 }
 
+/**
+ * ank_macro_hierarchy(arrow.typeclasses.MonadThrow)
+ *
+ * A MonadError with the error type fixed to Throwable. It provides [bindingCatch] for automatically catching throwable
+ * errors in the context of a binding, short-circuiting the complete computation and returning the error raised to the
+ * same computational context (through [raiseError]).
+ */
 interface MonadThrow<F> : MonadError<F, Throwable> {
 
   /**
-   * Entry point for monad bindings which enables for comprehensions. The underlying implementation is based on coroutines.
-   * A coroutine is initiated and suspended inside [MonadErrorContinuation] yielding to [Monad.flatMap]. Once all the flatMap binds are completed
-   * the underlying monad is returned from the act of executing the coroutine
+   * Entry point for monad bindings which enables for comprehensions. The underlying implementation is based on
+   * coroutines. A coroutine is initiated and suspended inside [MonadErrorContinuation] yielding to [Monad.flatMap].
+   * Once all the flatMap binds are completed, the underlying monad is returned from the act of executing the coroutine.
    *
-   * This one operates over [MonadError] instances that can support [Throwable] in their error type automatically lifting
-   * errors as failed computations in their monadic context and not letting exceptions thrown as the regular monad binding does.
+   * This one operates over [MonadError] instances that can support [Throwable] in their error type automatically
+   * lifting errors as failed computations in their monadic context and not letting exceptions thrown as the regular
+   * monad binding does.
```

Comment:
Can we add a playground example similar to those in Bracket and Functor that illustrates the usage of `bindingCatch` ?